samples/bookinfo/README.md

```diff
@@ -2,34 +2,97 @@
 
 See <https://istio.io/docs/examples/bookinfo/>
 
-## Build docker images without pushing
+## Build docker images
 
 ```bash
+cd samples/bookinfo
 src/build-services.sh <version> <prefix>
 ```
 
 Where `<version>` is the tag and `<prefix>` is the docker registry to tag the images.
 
-For example: `src/build-services.sh 1.1.0 docker.io/istio`.
+For example:
+
+```bash
+$ src/build-services.sh 1.16.3 docker.io/shamsher31
```

Comment:
This is fine for local testing, but we need to push the images to `docker.io/istio` before we can merge the updated bookinfo.yaml. What is the process for doing that, i.e., getting the images officially published at docker.io?

Reply:
I think official publishing requires special permission and I don't have idea about how that works. These steps are for building and testing with personal Docker account. 

Maybe @howardjohn can help?

Reply:
It's the same process only using docker.io/istio, instead of docker.io/userid, but the push needs to be done by somebody with owner permission to the istio dockerhub organization. We should say that in this readme, using your own docker.io org is just for local testing. Ideally we would have a build process that does the docker.io/istio push automatically.

Reply:
@frankbu and I and John (and others) have authority to build and push the final images to Docker. I'm not sure the list will stay the same if we move to gcr or somewhere else.

Reply:
But, we need the owner of a PR to to the appropriate testing with their own built/pushed images before we would build/push official ones, and then update to use them.

Reply:
Added note.